linux_os/guide/services/xwindows/disabling_xwindows/package_xorg-x11-server-common_removed/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_sle,multi_platform_rhv,multi_platform_fedora,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Oracle Linux 7,Oracle Linux 8
+
+# Install required packages
+{{{ bash_package_remove("xorg-x11-server-common") }}}
+
+systemctl set-default multi-user.target
```

Comment:
NACK. This snippet should only remove a package.

Reply:
reverted